src/h5web/dataset-visualizer/DatasetVisualizer.tsx

```diff
@@ -28,8 +23,8 @@ function DatasetVisualizer(props: Props): JSX.Element {
         onChange={setActiveVis}
       />
       <div className={styles.displayArea}>
-        {dataset && activeVis ? (
-          <VisDisplay vis={activeVis} dataset={dataset} />
+        {dataset ? (
+          activeVis && <VisDisplay vis={activeVis} dataset={dataset} />
```

Comment:
If `dataset` is defined but `activeVis` isn't, it means it's either the first render or a render where `activeVis` is stale. Either way, `activeVis` will be defined in the next render, so we don't need to show the "Nothing to visualize" message.